Scheduler/observing_blocks_maintel/BLOCK-365.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "Imaging",
+    "program": "BLOCK-365",
+    "constraints": [],
+    "scripts": [
+        {
+            "name": "maintel/track_target_and_take_image_lsstcam.py",
+            "standard": true,
+            "parameters": {
+                "ra": "$ra",
+                "dec": "$dec",
+                "rot_sky": "$rot_sky",
+                "band_filter": "$band_filter",
+                "name": "$name",
+                "estimated_slew_time": "$estimated_slew_time",
+                "exp_times": "$exp_times",
+                "num_exp": "$num_exp",
+                "obs_time": "$obs_time",
+                "program": "$program",
+                "reason": "science",
+                "targetid": "$targetid"
+            }
```

Comment:
I really need "scheduler_note" to get passed along to the consdb. I don't know how to do that .. does it need to get added to this BLOCK as well? (and then also grabbed from the FBS by ts_scheduler?).

Reply:
Lynne and I discussed this briefly offline, I think we can pass this to the `note` in the track_target_and_take_images script, which will then get written to the OBSANNOT value in the header. We will need a new ticket to add this configuration to the script and to make sure it is available from the Scheduler (already being worked into the next Scheduler CSC version).